logstreams/src/test/java/io/camunda/zeebe/logstreams/impl/log/LogStreamReaderTest.java

```diff
@@ -26,26 +26,21 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.rules.RuleChain;
-import org.junit.rules.TemporaryFolder;
 
 public final class LogStreamReaderTest {
   private static final UnsafeBuffer EVENT_VALUE = new UnsafeBuffer(getBytes("test"));
   private static final int LOG_SEGMENT_SIZE = (int) ByteValue.ofMegabytes(4);
 
   @Rule public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
:wrench: We can address the warning now - I think this is not used anywhere, so let's just remove it

Reply:
Interesting that intellij doesn't show any warning or hint here

Reply:
I guess because it's a rule, and rules are implicitly used. But technically as we never change the expectation, it sounds to me like it's useless? Maybe I misunderstood how the rule works? :thinking: